### DISCUSSION TITLE
Settings: Open with ... RT > close CMD windows (on Windows PC) - ?

### DISCUSSION THREAD

**Discussion by ghost** [Beep6581/RawTherapee#6719] [Ideas]
Hey, to not open an issue, i am allways using `open with ... RawTherapee` (or _double-click_ for standard app) and this opens a CMD window which

- calls the RT runtime
- logs informations
- pauses to close

I am comparing lots of images via RT and have an editor window open for every image, but this means a CMD window, too. And when closing an image, or several, the CMD(s) stay open. This is annoying as i do not need the information.

Is it possible to add an setting to choose that the CMD should stay open after opening an image in editor? I would like to only have the editor windows in my task-list.

**Comment by Lawrence37** [comment]
The `-w` command line option does that (see http://rawpedia.rawtherapee.com/Command-Line_Options#RawTherapee_GUI). To add the option to the Windows "Open with", you may need to [edit the registry](https://superuser.com/questions/361816/pass-command-line-arguments-to-windows-open-with).
